esp/esp/settings.py

```diff
@@ -115,3 +115,18 @@
 CSRF_COOKIE_NAME = 'esp_csrftoken'
 
 SKIP_SOUTH_TESTS = True # To disable South's own unit tests
+
+if SENTRY_DSN:
+    # If SENTRY_DSN is set, send errors to Sentry via the Raven exception
+    # handler. Note that our exception middleware (i.e., ESPErrorMiddleware
+    # and PrettyErrorEmailMiddlware) will remain enabled and will receive
+    # exceptions before Raven does.
+    import raven
+
+    INSTALLED_APPS += (
+        'raven.contrib.django.raven_compat',
+    )
+    RAVEN_CONFIG = {
+        'dsn': SENTRY_DSN,
+        'release': raven.fetch_git_sha(os.path.dirname(os.path.dirname(os.path.dirname(__file__)))),
```

Comment:
(Also, I think `PROJECT_ROOT` is already what you want, although at the moment it's set manually rather than with a relative path.)